tools/tasks/css_task.ts

```diff
@@ -1,9 +1,11 @@
 import { Task } from './task';
+import Config from '../config';
 
 export abstract class CssTask extends Task {
 
+    // Is it still usefull if cache used ?
   shallRun(files: String[]) {
-    return files.some(f =>
+    return Config.ENABLE_SCSS || files.some(f =>
```

Comment:
Yes it's exact. With the caching, no need to check is any .css or .scss changed.